### PR TITLE
Rpc Protos 0.205.0

### PR DESCRIPTION
**Pull request by SkOODaT**
Rpc Protos 0.205.0

## Description
Rpc Protos 0.205.0

## Motivation and Context
Rpc Protos 0.205.0

## How Has This Been Tested?
Locally on my own map, ive merged it into stock but it needs to be tested again as i dont run a stock rdm

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Furtif** [comment]
need merge https://github.com/123FLO321/POGOProtos-Swift/pull/8 nedd more ref repo @123FLO321

**Review by Furtif — DISMISSED** [review]

**Review by Furtif — DISMISSED** [review]

**Review by feigei — APPROVED** [review]
I have tested this with 2 devices and I am getting data without any issues.

**Review by ch1zzu — APPROVED** [review]

**Review by Furtif — APPROVED** [review]

**Review by Fabio1988 — COMMENTED** [review]
Have someone tested it?
On 4th may the rainy lure event is coming... so until this event it would be great to merge, because we need the data from this  RPC PROTOS

File: Sources/RealDeviceMap/Misc/POGOProtos_Inventory_Item_ItemId.swift [hunk 1]
```diff
@@ -8,68 +8,77 @@
...
-        //.itemLegendaryRaidTicket,
-        .itemStarPiece
-        //.itemFriendGiftBox,
-        //.itemTeamChange,
-        //.itemVsSeekerBattleNow,
-        //.itemLeaderMapFragment,
-        //.itemLeaderMap,
-        //.itemGiovanniMap,
-        //.itemGlobalEventTicket,
+    public static var allAvilable: [Item] = [
```
[the remark this answers is not in the document]
Comment (SkOODaT) [reply Sources/RealDeviceMap/Misc/POGOProtos_Inventory_Item_ItemId.swift]: im just copying what flos has 😉

[the remark this answers is not in the document]
Comment (Furtif) [reply Sources/RealDeviceMap/Misc/POGOProtos_Inventory_Item_ItemId.swift]: is not merged this still  ????

[the remark this answers is not in the document]
Comment (Fabio1988) [reply Sources/RealDeviceMap/Misc/POGOProtos_Inventory_Item_ItemId.swift]: > im just copying what flos has wink
> thiers so many spelling mistakes lol

yeah those spelling mistakes are fixed in #204 :)

**Pull request merged**